plugins/statistics.koplugin/main.lua

```diff
@@ -1163,6 +1199,27 @@ The max value ensures a page you stay on for a long time (because you fell aslee
                             },
                         },
                     },
+                    {
+                        text_func = function()
+                            local option
+                            local setting_value = self.settings.use_reference_pages
+
+                            if setting_value == 1 then
+                                option = _("When used for document")
+                            elseif setting_value == 2 then
+                                option = _("When available")
+                            else
+                                option = _("Never")
+                            end
+
+                            return T(_("Use reference pages: %1"), option)
+                        end,
```

Comment:
(Blank lines not needed in such little sections of code. Good for separating logical steps - but otherwise useless.)